pyccel/ast/numpyext.py

```diff
@@ -1185,11 +1186,11 @@ class NumpyArctanh(NumpyUfuncUnary) : pass
 
 #=======================================================================================
 
-class NumpyAbs(NumpyUfuncUnary):
+class NumpyAbsolute(NumpyUfuncUnary):
```

Comment:
What is the difference between `NumpyAbs` and `NumpyAbsolute`? Why do we need both classes?
According to https://numpy.org/doc/stable/reference/generated/numpy.absolute.html:
> numpy.absolute
> np.abs is a shorthand for this function.

Reply:
@rakati I have the same question here...

Reply:
I was getting a different return type from abs and absolute functions.
so as a solution I separate them. but after your comment, I did a small search and I found that abs and absolute is quite the same. I think I made a mistake somewhere, I will check that again.

Reply:
Maybe you were using `abs` and `numpy.absolute` rather than `numpy.abs` and `numpy.absolute`?

Reply:
I just realize the problem
```python
import numpy as np

type(np.fabs(479092))
# output >> numpy.float64

type(np.absolute(479092))
# output >> numpy.int64
```
I didn't pay attention that it is `fabs` function not `abs`.